src/components/ChannelSearch/SearchInput.tsx

```diff
@@ -50,7 +52,7 @@ export const SearchInput = <
       onChange={(event: React.BaseSyntheticEvent) =>
         searchFunction ? searchFunction(channelSearchParams, event) : onSearch(event)
       }
-      placeholder={t('Search')}
+      placeholder={placeholder || t('Search')}
```

Comment:
```suggestion
      placeholder={placeholder ?? t('Search')}
```
In case someone would want to disable the placeholder completely (passing down empty string).